tests/e2e/test_main.py

```diff
@@ -701,10 +701,21 @@ def test_upload_download_subdir(
 
 
 @pytest.fixture
-def args_data_cp_from_cloud(cli_runner: CLIRunner) -> Callable[..., List[str]]:
-    def _f(bucket: str, src: str, dst: str, extract: bool, compress: bool) -> List[str]:
+def args_data_cp(cli_runner: CLIRunner) -> Callable[..., List[str]]:
```

Comment:
disagree with renaming `args_data_cp_from_cloud` -> `args_data_cp`. This fixture doesn't serve for `args_data_cp_to_cloud`

Reply:
What would you suggest? Keep as it was?

Reply:
revert renaming to `args_data_cp_from_cloud`. But you're right it's very minor